flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SpillableSubpartitionTest.java

```diff
@@ -300,6 +315,230 @@ public void testConsumeSpillablePartitionSpilledDuringConsume() throws Exception
 		assertTrue(buffer.isRecycled());
 	}
 
+	/**
```

Comment:
could some of those tests be squashed into fewer methods? Or you think that wouldn't be a good idea?

Reply:
you're right - some tests actually share most of their code. I'll extract a common test method to reduce some code.